docs/index.asciidoc

```diff
@@ -2,6 +2,10 @@
 :server-branch: 6.5
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 
+:uri-string: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type
```

Comment:
What if we build the links into the attributes to make referencing them even easier? So, something like this:
```suggestion
:uri-string: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type[<string>]
```

Then, in the API docs, we reference them like this: `+{uri-string}+`. Saves us from having to put the entire link in the docs over and over.

Reply:
I like! This is clever